libs/SalesforceSDK/src/com/salesforce/androidsdk/rest/RestRequest.java

```diff
@@ -114,26 +143,96 @@ public String getPath(Object... args) {
 	private final String path;
 	private final RequestBody requestBody;
 	private final Map<String, String> additionalHttpHeaders;
-	
```

Comment:
For batch and composite, we are given a collection of RestRequest and we need to have a JSON representation for them.